ocs_ci/ocs/resources/mcg.py

```diff
@@ -693,7 +693,12 @@ def cli_create_bucketclass(self, name, backingstores, placement):
         """
         backingstore_name_list = [backingstore.name for backingstore in backingstores]
         bc = f" --backingstores={','.join(backingstore_name_list)} --placement={placement}"
-        self.exec_mcg_cmd(f"bucketclass create {name}{bc}")
+        placement_parameter = (
+            "placement-bucketclass "
```

Comment:
should we save "placement-bucketclass" in constant.py?

Reply:
It's only ever used once, in this instance (placement bucketclass creation), so personally, I think that putting it in a constant might be a bit excessive

Reply:
I still think it's better to put it there, so in case we will need to use this in the future in other places it will be available